### PR TITLE
feat: Add json and markdown format to --run-statistics

### PR DESCRIPTION
**Pull request by amandahla**
Proposal: add formats json and markdown to statistics output

Why: https://github.com/aquasecurity/tfsec/issues/1790

### CODE REVIEW

File: pkg/scanners/terraform/executor/statistics.go [hunk 1]
```diff
@@ -12,27 +14,54 @@ import (
...
 	sorted := SortStatistics(statistics)
+
+	if format == "json" {
+		result := StatisticsResult{Result: sorted}
+		val, err := json.MarshalIndent(result, "", "    ")
+		if err != nil {
+			return err
+		}
+
+		fmt.Println(string(val))
```
Comment (liamg) [inline pkg/scanners/terraform/executor/statistics.go:51]: We have a writer to send the output to here, i.e.
```suggestion
		_, _ = fmt.Fprintln(w, string(val))
```

**Review by liamg — CHANGES_REQUESTED** [review]
Thanks @amandahla this is great - I've added one minor comment then we're good to merge :+1:

**Review by liamg — APPROVED** [review]

**Comment by liamg** [comment]
I've updated the branch with the minor change, hope this is ok :)

**Comment by amandahla** [comment]
> I've updated the branch with the minor change, hope this is ok :)

Thanks @liamg , sorry for the delay!

**Pull request merged**